llvm/cmake/modules/LLVMExternalProjectUtils.cmake

```diff
@@ -349,6 +349,8 @@ function(llvm_ExternalProject_Add name source_dir)
                -DPACKAGE_VERSION=${PACKAGE_VERSION}
                -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
                -DCMAKE_MAKE_PROGRAM=${CMAKE_MAKE_PROGRAM}
+               -DLLVM_ENABLE_SPHINX=${LLVM_ENABLE_SPHINX}
+               -DLLVM_BUILD_DOCS=${LLVM_BUILD_DOCS}
```

Comment:
Why are these changes needed? If needed, I think it would be better to do this in a separate review.

Reply:
Dang! I am clearly `git` stupid! Yes, these are unrelated. I will remove! Sorry!